modules/naive-ui.yml

```diff
@@ -0,0 +1,20 @@
+
+name: Naive UI
+description: >-
+  A Vue 3 Component Library
+  Fairly Complete, Theme Customizable, Uses TypeScript, Fast
+  Kinda Interesting
```

Comment:
Im actually not sure if this fits in the description ^^

Reply:
Thank you i will check soon